src/sonic-config-engine/minigraph.py

```diff
@@ -2241,7 +2241,7 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
     # #voq switch_id for asic
     # switch_id = chassis_metadata.get(asic_hostname, {}).get('asic_switch_id', None)
     # on Voq system each asic has a switch_id
-    if switch_id is not None:
+    if switch_id is not None and chassis_type == CHASSIS_CARD_VOQ:
```

Comment:
can we change this to `chassis_type != CHASSIS_CARD_PACKET_CHASSIS`? in line 2248 for fabric cards we get the switch_id from minigraph and save it for `CHASSIS_CARD_FABRIC`